draft-ietf-shmoo-remote-fee.md

```diff
@@ -75,9 +75,8 @@ registration fee can be a barrier to participation.
 
 # Principle of open participation
 
-This document outlines the principle of open participation and solicits community feedback in
-order to reach consensus on this or a similar principle that the IETF Administration LLC (IETF LLC)
-can use to guide future decision about registration fees for full online meetings.
+This document outlines the principle of open participation that the IETF Administration LLC (IETF LLC)
+is expected to incorporate in decisions about registration fees for full online meetings.
```

Comment:
```suggestion
is expected to incorporate into decisions about the registration fee structure for fully online meetings.
```

I'll note that there remain some internal inconsistencies int he document that need to be addressed, for example, the very next paragraph says "regardless of whether the meeting has a physical presence", which implies the principles intend to hybrid meetings, which as discussed on the list they can't (at least not at BCP-level strength.) And further down it says "if remote participation is provided, there should always be a free option", which has similar issues.

Reply:
I copied you comment to PR #21 to address there!